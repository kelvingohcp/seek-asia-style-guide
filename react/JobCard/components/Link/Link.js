import React from 'react';
import PropTypes from 'prop-types';

const Link = ({ link, children, ...restProps }) => {
  return (<a href={link} {...restProps}>{children}</a>);
};

export default Link;
Link.propTypes = {
  link: PropTypes.string,
  children: PropTypes.node
};
