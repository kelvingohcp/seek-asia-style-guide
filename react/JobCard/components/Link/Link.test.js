import React from 'react';
import { shallow } from 'enzyme';

import Link from './Link';

describe('JobCard - Link', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Link link={'/JobCard'} >link</Link>);
    expect(wrapper).toMatchSnapshot();
  });
});
