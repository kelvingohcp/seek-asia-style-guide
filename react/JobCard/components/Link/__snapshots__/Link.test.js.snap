// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JobCard - Link should render correctly 1`] = `
<a
  href="/JobCard"
>
  link
</a>
`;
